.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+## Scope 
+Description of feature / issue, and links to issue...
```

Comment:
I quite like to give a scope - even if it is just a super concise summary of the goals, or a reference to the issue.